env/staging/app/terragrunt.hcl

```diff
@@ -167,9 +168,9 @@ inputs = {
   sns_topic_alert_warning_arn  = dependency.sns.outputs.sns_topic_alert_warning_arn
   sns_topic_alert_ok_arn       = dependency.sns.outputs.sns_topic_alert_ok_arn
 
-  cognito_endpoint_url      = dependency.cognito.outputs.cognito_endpoint_url
-  cognito_client_id         = dependency.cognito.outputs.cognito_client_id
-  cognito_user_pool_arn     = dependency.cognito.outputs.cognito_user_pool_arn
+  cognito_endpoint_url  = dependency.cognito.outputs.cognito_endpoint_url
+  cognito_client_id     = dependency.cognito.outputs.cognito_client_id
+  cognito_user_pool_arn = dependency.cognito.outputs.cognito_user_pool_arn
```

Comment:
I think there is a missing declaration here:
```
cognito_user_pool_id = dependency.cognito.outputs.cognito_user_pool_id
```

Reply:
It was an output that I had created but no longer used. Will remove it